apps/web/src/views/Farms/v4/components/PoolsPage.tsx

```diff
@@ -9,16 +10,23 @@ import {
   SubMenu,
   ITableViewProps,
   TableView,
+  FeeTier,
 } from '@pancakeswap/uikit'
+import { TokenPairImage } from 'components/TokenImage'
 import { useTranslation } from '@pancakeswap/localization'
+import { ERC20Token } from '@pancakeswap/sdk'
+import { TokenOverview } from '@pancakeswap/widgets-internal'
+import { useFarmsV3WithPositionsAndBooster } from 'state/farmsV3/hooks'
 import { PoolsFilterPanel } from './PoolsFilterPanel'
 
 interface IDataType {
   name: string
-  feeTier: number
-  apr: number
-  tvl: number
+  feeAmount: number
+  cakeApr: number
```

Comment:
just apr is ok I think, cakeApr is farmApr, there are others apr like lpFeeApr

Reply:
same as above